src/app/lib/utilities/moment/index.stories.jsx

```diff
@@ -10,6 +10,7 @@ import {
   GEL_SPACING_DBL,
 } from '@bbc/gel-foundations/spacings';
 import { GEL_FF_REITH_SANS } from '@bbc/gel-foundations/typography';
+import WithTimeMachine from '../../../../testHelpers/withTimeMachine';
```

Comment:
Does the alias work here?
```suggestion
import WithTimeMachine from '#testHelpers/withTimeMachine';
```

Reply:
Yes.